Src/StartMenu/StartMenuDLL/StartMenuDLL.cpp

```diff
@@ -687,6 +687,15 @@ bool PointAroundStartButton( size_t taskbarId, const CPoint &pt )
 		return pt.x<rc.right;
 }
 
+SIZE* GetTaskBarSize(size_t taskbarId)
+{
+	const TaskbarInfo* taskBar = GetTaskbarInfo(taskbarId);
+	RECT rcTask;
+	GetWindowRect(taskBar->taskBar, &rcTask);
+	SIZE *taskbarSize = new SIZE{ rcTask.right - rcTask.left,rcTask.bottom - rcTask.top };
+	return taskbarSize;
```

Comment:
`return { rcTask.right - rcTask.left, rcTask.bottom - rcTask.top };` would be enough once the return value is `SIZE`.